app/src/main/resources/db/migration/V4__add_service_definitions.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE IF NOT EXISTS service_definitions (
+    id BIGINT PRIMARY KEY AUTO_INCREMENT,
+    service_id BIGINT NOT NULL,
+    FOREIGN KEY (service_id) REFERENCES services(id)
+);
+
+CREATE TABLE IF NOT EXISTS service_definition_attributes (
+    id BIGINT PRIMARY KEY AUTO_INCREMENT,
+    attribute_order INTEGER NOT NULL,
+    code VARCHAR(255),
+    datatype VARCHAR(255),
+    datatype_description VARCHAR(255),
+    description VARCHAR(255),
+    required BIT NOT NULL,
+    variable BIT NOT NULL,
+    service_definition_id BIGINT NOT NULL,
+    FOREIGN KEY (service_definition_id) REFERENCES service_definitions(id)
+);
+
+CREATE TABLE IF NOT EXISTS service_definition_attribute_values (
+    id BIGINT PRIMARY KEY AUTO_INCREMENT,
+    value_name VARCHAR(255),
+    service_definition_attribute_id BIGINT NOT NULL,
+    FOREIGN KEY (service_definition_attribute_id) REFERENCES service_definition_attributes(id)
+);
```

Comment:
I think we should drop the two service_definition_json and metadata columns from the service table as part of this pr

Reply:
Ahh, I knew I forgot something! Thanks, I'll make the update.